CMSgov/easi-app
# [EASI-2113] Handle legacy intakes requester

# EASI-2113

## Changes and Description

Fixes bug with legacy system intakes requester
- Previous work on 2113 added requester as contact in system intake form, so legacy intake requesters do not have an ID
- This was causing "unverified recipients" alert to show even when all recipients could be selected
- Fixed alert conditional so that it ignores requester

## How to test this change

- Sign in locally as valid EUA ID
- Comment out `updateSystemIntakeContact('requester');` in [src/views/SystemIntake/ContactDetails/index.tsx](https://github.com/CMSgov/easi-app/blob/af0960e59d61fae14fa3e99ccdf5dc74be01a94c/src/views/SystemIntake/ContactDetails/index.tsx#L152)
  - This will create a system intake that does NOT add the requester as a contact
- Create system intake request 
- Complete any action on the created intake request
  - Action screen should not show unverified recipients alert

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
